.github/workflows/tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ] # , macos-latest, windows-latest ]
```

Comment:
We might want to add those other operating systems to the build matrix within a later iteration.